projects/robots/universal_robots/resources/ros_package/ur_e_webots/README.md

```diff
@@ -1,6 +1,22 @@
+## Installation
+
+To use the package you need to copy it in your catkin workspace, install the dependencies and compile it:
+```shell
+cd /path/to/catkin_ws/src
+cp $WEBOTS_HOME/projects/robots/universal_robots/resources/ros_package/ur_e_webots .
+cd ..
+rosdep update
+rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO
+catkin_make
+source devel/setup.bash
+```
+
+You may also install the `universal_robot` optional package to get the URDF definitions of the robots (used for example for RVIZ visualization).
```

Comment:
Isn't the ros package name `universal_robots`?
```suggestion
You may also install the `universal_robots` optional package to get the URDF definitions of the robots (used for example for RVIZ visualization).
```

Reply:
Actually, they support both names, but the root one is without 's':
https://github.com/ros/rosdistro/blob/8d9d7cfa6ef2fddac296be705e7ad61554fa41b0/kinetic/distribution.yaml#L16513

Reply:
Ok.
On the ROS wiki the only page is named `universal_robots` (redirected from `universal_robot`):
http://wiki.ros.org/action/show/universal_robots?action=show&redirect=universal_robot

Reply:
Ok, then it is indeed probably better to stick to the officially documented name (they don't seem very coherent with which name).